com.unity.formats.fbx/Editor/FbxExporter.cs

```diff
@@ -2790,6 +2790,7 @@ private FbxNode CreateFbxNode(GameObject unityGo, FbxScene fbxScene)
             string fbxName = unityGo.name;
             if (ExportOptions.UseMayaCompatibleNames)
             {
+                Undo.RecordObject(unityGo, "rename " + fbxName);
```

Comment:
could this be placed in the `if (ExportOptions.AllowSceneModification)` a couple of lines below?
The scene will only be modified if the option is true.